requirements.txt

```diff
@@ -1,7 +1,7 @@
 Django==1.9
 mysql-python==1.2.5
 python-ldap==2.4.27
-requests==2.11.1
+requests==2.20
```

Comment:
Can we use the version style as the one in student-dashboard-django project, and mark this as 

requests>=2.20,<2.20.99

Reply:
Absolutely! 
Do we want to change the nomenclature of all the dependencies in the requirements.txt file to assume this broad match for version control?

Reply:
Let's do the request change only this time, to address the security alert.

I think you can leave the change as "requests==2.20" for now, and open a separate issue for rewrite the all dependencies with new nomenclature format. Sorry about the confusion.